application/features/notification.feature

```diff
@@ -6,8 +6,29 @@ Feature: Sending notifications
     When I call the sendMissingEmailNotice function
     Then an email is sent
 
+  Scenario: Don't send missing email notification
+    Given a specific user exists in the ID Store
+      But the user does not exist in the ID Broker
+    When I get the user info from the ID Store and send it to the ID Broker
+    Then an email with subject "Email address missing" is not sent
```

Comment:
This scenario seems to leave something implied that seems important to its statement about what the application does. Is it important, for instance, that the user info obtained from the ID Store (and that the application will try to send to the ID Broker) lacks an email address?

Reply:
I guess I'm just confused on why this test was added, and what it's proving about the code.

Reply:
It's the "normal" case, proving that you won't send a "missing email" message for every new user. I'll see if I can reword it to help with that understanding.

Reply:
Ahh, thanks. Maybe just rewording the title of the scenario would accomplish that.